.yamlfmt.yaml

```diff
@@ -2,6 +2,7 @@ continue_on_error: true
 match_type: doublestar
 exclude:
 - '**/zz_fixture_*.yaml'
+- 'secret-sync-controller/deploy/templates/*'
```

Comment:
@geoberle  I am surprised you had to do this - `yamlfmt` will just ignore any file that isn't valid YAML (e.g. has template braces in there)

Reply:
forget about it. it was a stupid attempt. fixed by adapting the unwrap script. take a look there